dm/proto/dmmaster.proto

```diff
@@ -302,3 +305,13 @@ message RegisterWorkerResponse {
     bool result = 1;
     string msg = 2;
 }
+
+message OfflineWorkerRequest {
+    string name = 1;
+    string address = 2;
```

Comment:
1. no _dmctl_ command added for `OfflineWorker`?
2. Is it safe for users to specify both of them (`name` and `address`) at the same time? Is it fine for only `name`?

Reply:
No. I check it and require the address and name of worker must match.